ui/client/components/SummaryIntroDialog.js

```diff
@@ -398,7 +400,7 @@ const SummaryIntroDialog = ({
                     type="submit"
```

Comment:
Quick question on the PR:

Is the reason that the default event page reload was occurring in the first place because the buttons were wrapped in a `<form>`? Is wrapping in a form, or using the `type=submit` prop necessary, then?

Basically an observation, that if not wrapping in form or not attaching a type=submit would also fix the problem, that might be a more direct fix than stopping the event propagation. At least to me, it is clearer that we don't wish to submit a form and can potentially prevent confusion/regressions.

Just a suggestion, but I think as it stands is good enough, and I might be missing something.

Reply:
I added the <form> wrapper and type=submit a long while back because I thought it made for a better experience to be able to type the model name (the step you need to do to confirm you want to version bump) and then hit return/enter, instead of needing to click with your mouse. This probably matters more to me as the person who does it many times in a row developing the page than any real user, so it probably doesn't matter much either way. It isn't a bug fix that comes with any real downsides, however, so I think the marginal benefit in UX of being able to submit with enter is still worth it. 

Good question!

Reply:
And it is a form, of sorts - just one that only has a single text field that we aren't saving any information from. We just confirm that their input matches the model name before enabling the submit button.